tests/state_test.py

```diff
@@ -1826,6 +1826,10 @@ def ref(x):
       y2 = random.normal(jax.random.clone(k1), y.shape)
       self.assertAllClose(impl_vjp(t), ref_vjp(t))
 
+      if jax.config.read("jax_skip_slow_tests"):
```

Comment:
`jtu._SKIP_SLOW_TESTS.value`

Although I would also rename `_SKIP_SLOW_TESTS` to `SKIP_SLOW_TESTS` if we're going to use it outside its enclosing module.